android/PhysicalWeb/app/src/main/res/values/strings.xml

```diff
@@ -49,6 +49,11 @@
     <string name="prod_pws">Production Open Source Physical Web Service</string>
     <string name="dev_pws">Development Open Source Physical Web Service</string>
     <string name="custom_pws">Custom Physical Web Service</string>
+    <string name="scan_settings_key">scan_settings</string>
```

Comment:
Only human readable strings should go in here...just make this a private static value